CharacterCreation/02_Classes.txt

```diff
@@ -33,10 +31,10 @@ Class Feats:
 	V
 	Maximum of 6 activations per encounter; costs [20 Nanites] to use.
 	----------------------
-	Battle Cry / Raise Moral -
-	You roar as you lead the charge against the enemy. 1 Action. 50% + (CHA mod) chance to 
-	raise allied effectiveness 20% (or +2 if relevant) on each of their next rolls [including 
-	your next roll]. You must use your next action to run towards enemies [minimum 1m]. 
+	Battle Cry / Raise Morale -
+	You roar to incite courage in your allies. 1 Action. 50% + (CHA mod) chance to 
+	raise allied effectiveness 20% (or +2 if relevant) on each of their next rolls 
+	[including your next roll]. 
```

Comment:
Great clarification! Thank you!

Reply:
I appreciate the compliment, but I changed how Battle Cry works, rather than clarified it. Are you talking about something else?

Reply:
Ah you must be referring to the roar itself

Reply:
my bad. I was reading fast and thought [your own roll too] was added by you.

Reply:
no worries! :)